sncosmo/models.py

```diff
@@ -151,8 +151,10 @@ def _bandflux(model, band, time_or_phase, zp, zpsys):
     for b in set(band):
         mask = band == b
         b = get_bandpass(b)
-
-        fsum = _bandflux_single(model, b, time_or_phase[mask])
+        try:
+            fsum = _bandflux_single(model, b, time_or_phase[mask])
+        except ValueError:
+            continue
```

Comment:
Why can we just ignore fsum in certain cases? Do we need code in the except block or a comment on why we can ignore fsum in this case?

Reply:
Apologies, @benjaminrose, I initially replied to this via email but should have responded here.

The issue was a leftover from an old patch that was mistakenly pushed.
I’ve reverted it.